### ISSUE TITLE
[BUG] Example not working. 

### ISSUE THREAD

**Issue by ajnart** [creyD/prettier_action#117]
I've just set up the Action with the following example in the README
```yaml
name: Prettier CI

on:
  pull_request:
    branches: [main]

jobs:
  prettier:
    runs-on: ubuntu-latest

    steps:
      - name: Checkout
        uses: actions/checkout@v3
        with:
          # Make sure the actual branch is checked out when running on pull requests
          ref: ${{ github.head_ref }}
          # This is important to fetch the changes to the previous commit
          fetch-depth: 0

      - name: Prettify code
        uses: creyD/prettier_action@v4.2
        with:
          # This part is also where you can pass other options, for example:
          prettier_options: --write **/*.{js,md}
          only_changed: True
```
**It simply doesn't work.**
I get ```
/home/runner/work/_actions/creyD/prettier_action/v4.2/entrypoint.sh: line 74: prettier: command not found
Problem running prettier with --write **/*.{js,md}
Error: Process completed with exit code 1.
```

**Comment by creyD** [comment]
Can you please try the same with v4.3?

**Comment by ajnart** [comment]
> Can you please try the same with v4.3?

I think I already did and it got me the same error

**Comment by jdelStrother** [comment]
FWIW I ran into this on 4.2 (because I just copy-and-pasted from the [example code](https://github.com/marketplace/actions/prettier-action#example-1-run-on-push-in-branch-main)). However, it worked fine for me in 4.3.

**Comment by creyD** [comment]
@ajnart Closing this for now, as it was a know issue on 4.2 and was fixed with 4.3
